cmd/quickstart.go

```diff
@@ -184,12 +196,18 @@ func quickstartExec(ctx context.Context, flags QuickstartFlags) error {
 		quickstartObj.WorkflowFile.Sources[sourceName].Inputs[0].Location = referencePath
 	}
 
-	speakeasyFolderPath := outDir + "/" + ".speakeasy"
-	if _, err := os.Stat(speakeasyFolderPath); os.IsNotExist(err) {
-		err = os.MkdirAll(speakeasyFolderPath, 0o755)
+	if quickstartObj.IsUsingSampleOpenAPISpec {
+		absSchemaPath := filepath.Join(outDir, "openapi.yaml")
+		if err := os.WriteFile(absSchemaPath, []byte(sampleSpec), 0o644); err != nil {
+			return errors.Wrapf(err, "failed to write sample openapi spec")
+		}
+		fmt.Printf("The OpenAPI sample document will be used.\nIt can be found here, you can edit it at anytime:\n\n  %s\n\n", absSchemaPath)
```

Comment:
@ryan-timothy-albert I feel it's important to show them explicitly the location of the file so they can edit afterwards. I'm just using print here, would you recommend using charm for this? I don't want this message to be cleared by the visualization.

Reply:
At least style it with charm. Maybe use an info style. Look at charm/styles.go for this

Reply:
![image](https://github.com/speakeasy-api/speakeasy/assets/1690659/82c5b96d-1d7d-467c-a53d-062c26598553)

Does it feel a bit much?

Reply:
Nope I like it